### PR TITLE
docs: Add caution about root user credentials

### PR DESCRIPTION
**Pull request by tam0ri**
Addresses #1690

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: site/content/docs/credentials.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ Copilot uses the AWS credentials from the [default credential provider chain](ht
 !!! tip
     We **recommend using a [named profile](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html)** to store your application's credentials. 
 
+!!! caution
+    You **can not** use AWS account root user credentials for Copilot.
```
Comment (iamhopaul123) [inline site/content/docs/credentials.md]:
```suggestion
    You **cannot** use AWS account root user credentials for Copilot.
```

**Review by iamhopaul123 — APPROVED** [review]
LGTM! Thank you!

File: site/content/docs/credentials.md (hunk 1 above)
Comment (efekarakus) [reply site/content/docs/credentials.md]: I feel like `!!!caution` is a bit excessive to put at the top for this scenario, given that it's discouraged by IAM to use access keys of the root user so I don't think it's very common. I think maybe instead at l.22 we can add:


> !!!note 
    You cannot use the AWS account root user credentials for your application. Please first create an IAM user instead as described [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html).

**Review by efekarakus — COMMENTED** [review]
Yay thank you so much for updating the docs!

**Review by efekarakus — APPROVED** [review]

**Pull request merged**